routes/middleware/jwt/issueToken.ts

```diff
@@ -6,7 +6,7 @@ import User from '../../../database/models/user';
 const issueToken = (req: Request, res: Response, next: NextFunction) => {
   const user: User = res.locals.user;
   const tokenSecret = process.env.TOKEN_SECRET;
-  const token = jwt.sign({ token: user.userID }, tokenSecret);
+  const token = jwt.sign({ token: user.pk }, tokenSecret);
```

Comment:
```suggestion
  const token = jwt.sign({ pk: user.pk }, tokenSecret);
```
token보다는 pk key값이 더 어울리는 거 같음